test/src/integration/integration_test.go

```diff
@@ -30,7 +30,6 @@ func TestEndToEndTerraform(t *testing.T) {
 		for _, m := range scalarModules {
 			terraformOptions := &terraform.Options{
 				TerraformDir: *terraformDir + *cloudProvider + "/" + m,
-				Vars:         map[string]interface{}{},
```

Comment:
Is this change related to this PR?
If not, please create another PR for that.

Reply:
I will thank you